### ISSUE TITLE
Arweave API gateway has changed.

### ISSUE THREAD

**Issue by rts-gordon** [luckyr13/arwiki#8]
Hi there, 
 Can't access arwiki.wiki from this Tuesday(7/3/2023), the API gateway has changed from arweave.net to ar-io.net, would you like to update the api gateway? Thank you.

**Comment by luckyr13** [comment]
Hello @rts-gordon , thanks for reporting this.

I have tested our arwiki.wiki domain but I can actually access normally. Perhaps could be a temporary problem? here are other domains that belong to us that you can give it a try if you like:

https://arwiki.wiki
https://wiki.g8way.io
https://arwiki.g8way.io

Can you please provide us with more information about the issue you are experiencing? I'm here to help, sorry for taking so long to respond and thanks for reaching us.

**Comment by rts-gordon** [comment]
Hi @luckyr13 
  Thanks for your reply.

  Maybe it is not a arwiki.wiki issue, but a arweave.net or ISP issue.

   I tested and found: if the home/office network via wireless like 4G/5G, we can't access arweave.net, or a mobile phone via 4G/5G; but if via wired network, we can access arweave.net.  But ar-io.net is available both.

  After arweave 2.6, the new gateway ar-io.net is available, does arwiki.wiki plan to change to new gateway?

**Comment by luckyr13** [comment]
hello again!

I asked on the `Arweave Dev` Discord channel and they told me that the Arweave API gateway is not gonna be replaced but there have been some issues with ISPs blocking arweave.net DNS records.

They also told me that ar-io.net gw points to the same gateway but avoids the problem of being blocked by DNS.

Anyway, we will try to stay up to date if any change happens. 

Also, just in case of emergency, if you can't have access from our main domain: https://arwiki.wiki please check our CHANGELOG file to get the list of all our Arwiki versions deployed: https://github.com/luckyr13/arwiki/blob/master/CHANGELOG.md

Latest version is 1.9.4: https://arweave.net/3grbkF0_puAQbeWaUF6SZrXn1qrhp2EpW1vkWNWvGqc

The important part is the tx ... just switch to the ar-io.net gw: https://ar-io.net/3grbkF0_puAQbeWaUF6SZrXn1qrhp2EpW1vkWNWvGqc

TX: `3grbkF0_puAQbeWaUF6SZrXn1qrhp2EpW1vkWNWvGqc`

But only in case of emergency haha or try with our other domains:
https://wiki.g8way.io/
https://arwiki.g8way.io/

Thanks for reporting this behavior, we will do our best to add an option in ArWiki for switching the gw, because yea, we use `arweave.net` as our default API gw and probably would be nice to have an option for selecting a different gw.

Do you agree that we close this issue now? Please let me know if you need something else and thanks for taking the time for reaching us.

**Comment by rts-gordon** [comment]
Thanks again.
Waiting for fix the arweave.net ISP issues.
